### ISSUE TITLE
3.3.2 release

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#1084]
Can be labelled **ready** when all other issues on this milestone are closed.

- [x] run code analysis in VS in *Release* mode and address violations (send a regular PR which must be merged before continuing)
- [x] if necessary, change `VERSION_SUFFIX` in [AppVeyor](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/settings) to  ""
- [x] create release-x.y.z branch for last-minute tasks
   - [x] create a pull request to the release-x.y.z branch that changes the version in CommonAssemblyInfo.cs to the expected version (of form _xx.yy.zz_)
- [x] submit a PR to merge the release-x.y.z branch to master
- [x] check [build](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/history)
- edit draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases):
    - [x] ensure completeness of release notes, including non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
    - [x] attach main nupkg and/or analyzer nupkg, whichever have content to release
    - [x] publish the release
- [x] push nupkgs to NuGet using the Deploy option on the AppVeyor build
- [x] de-list pre-release or superseded buggy NuGet packages if present
- [x] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [x] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [ ] post a link to the GitHub Release in each issue in this milestone, with thanks to contributors
- [x] create a new draft GitHub Release
- [x] create a new milestone for the next release
- [x] create a new issue (like this one) for the next release, adding it to the new milestone
- [x] close the milestone for this release
- [x] close this issue

**Comment by thomaslevesque** [comment]
https://twitter.com/fakeiteasyfx/status/860231402775879683
